### DISCUSSION TITLE
Release info gone, and latest release is incorrect

### DISCUSSION THREAD

**Discussion by innobead** [community/community#16316] [Other Feature Feedback, Questions, & Ideas] [answered]
https://github.com/longhorn/longhorn/releases the latest release is wrong, it should be 1.2.4. Also, many releases after that are all gone, but I still can search them via https://github.com/longhorn/longhorn/releases/tag/<version>. 

It just happened today. Need some fix. Thanks.

**Comment by innobead** [comment]
Any solution to fix that? also, what's the cause? Thanks.

**Comment by cp2004** [comment]
I created a (private) ticket yesterday, GitHub have acknowledged the problem and are working to fix it.

The workaround to make the releases show up again on the list:

> Editing the release and **clicking on Update release** without making any changes should force a re-sync and the missing release will re-appear.

**Reply by innobead** [reply]
exactly, but it seems you need to update each release :disappointed:

**Comment by canuckjacq** [comment, marked as answer]
Hi  👋🏻 

We have pushed a fix for this - can both of you check to see how your releases look now?

**Reply by innobead** [reply]
it works. thank you!

**Comment by finolacahill** [comment]
Hello. Still issues here IMO:
"Latest" release not updating (see time stamps):
![image](https://user-images.githubusercontent.com/45110160/183091203-6257e27d-50ee-4de4-b0ea-cda357db7f77.png)

**Reply by dt203344** [reply]
Got the same issue on GHE 3.7
